### ISSUE TITLE
Cannot select Free Features via Keyboard

### ISSUE THREAD

**Issue by AashikP** [woocommerce/woocommerce-admin#7131]
**Describe the bug**

On this step of onboarding, you cannot select the Free Features with just your Keyboard. You can only either install them all, or choose to not install any.

**To Reproduce**

1. Start Onboarding workflow - WooCommerce
2. Notice that when you are in the `Free Features` stage, you can't select the extensions you want to install unless you use a mouse

**Expected behavior**

Selection of Free Features need to be accessible via Keyboard.

**Screenshots**

![https://d.pr/i/3t4Hud+](https://d.pr/i/3t4Hud+)

**Desktop (please complete the following information):**

* OS: MacOS
* Browser : Chrome 
* Version : 91.0.4472.77
